scripts/ci/run_ci.sh

```diff
@@ -96,9 +96,18 @@ if [ -n "$MAIN_CI" ]; then
 	fi
 	source zephyr-env.sh
 	SANITYCHECK="${ZEPHYR_BASE}/scripts/sanitycheck"
+
+	# Possibly the only record of what exact version is being tested:
+	short_git_log='git log -n 5 --oneline --decorate --abbrev=12 '
+
 	if [ -n "$PULL_REQUEST_NR" ]; then
+		$short_git_log $REMOTE/${BRANCH}
+		# Now let's pray this script is being run from a
+		# different location
+# https://stackoverflow.com/questions/3398258/edit-shell-script-while-its-running
```

Comment:
Was this unindented to work around some CI line length check? Otherwise, I'd just let the line be as long as it needs.

Reply:
No opinion as to alignment or whether or not the joke belongs in the source, but can I say that's the most hilariously wrong accepted answer I've ever seen on SO.

Reply:
> Was this unindented to work around some CI line length check?

It was partly trying to dodge some gitlint or other bullet (which... doesn't actually exist, just checked) and partly not to write too far right: this sort of check exists for a reason! I don't really mind either way and will indent.

@andyross : I was also very amused by that SO answer, however I'm sorry it and/or my phrasing seemed to eclipse the "value" I tried to add with this comment: I once lost hair before finally understanding that the (run-time copy of the) shell script I was trying to debug was being modified while it was running. A comment like this one could have saved some of that hair.
Any rephrasing suggestion appreciated.